client/src/redux/project/project.sagas.ts

```diff
@@ -19,9 +22,11 @@ function* attemptCreateProject({ payload }: ProjectActionTypes) {
 		});
 
 		// Throws if the action was unsuccessful
-		// TODO Make this a user visible error
-		// FIXME 201 Doesn't nessecarily mean that the user failed to authentcate, it could be othrer things too
-		if (res.status !== 201) throw new Error('Failed to authenticate user');
+		// TODO Read the error description and solution from the response into the error message.
+		if (res.status !== 201) {
+			yield put(openError('Error', 'Here is a solution'));
```

Comment:
The error should take the description and solution from the response. But this doesn't exist yet. This will be  fixed in #31